docs/manual/macros.md

```diff
@@ -133,10 +133,10 @@ Sometimes it is useful to test whether a macro is defined or not. Syntax
 %{?!macro_name:value}
 ```
 
-can be used for this purpose. %{?macro_name:value} is expanded to "value"
+can be used for this purpose. `%{?macro_name:value}` is expanded to "value"
 if "macro_name" is defined, otherwise it is expanded to the empty string.
-%{?!macro_name:value} is the negative variant. It is expanded to "value" if
-"macro_name" not is defined. Otherwise it is expanded to the empty string.
+`%{?!macro_name:value}` is the negative variant. It is expanded to "value"
+if "macro_name" not is defined. Otherwise it is expanded to the empty string.
```

Comment:
Here, by the same logic, "value" wouldn't be fenced -- but `macro_name` probably should be. (Also, I just noticed the "not is defined" there.)
```suggestion
`%{?!macro_name:value}` is the negative variant. It is expanded to "value"
if `macro_name` is not defined. Otherwise it is expanded to the empty string.
```